v3/src/utilities/translation/lang/en-US.json5

```diff
@@ -734,6 +734,7 @@
 
     // DG.CellAxis
     "DG.CellAxis.other": "OTHER",
+    "DG.CellAxis.dragCategory": "Click and drag to reorder categories",
```

Comment:
You're more familiar than I with this process. But it seems to me we might as well get new strings into POEditor sooner rather than later. Perhaps there is a way to mark them there such that translators will not be wondering why they can't find the string in V2?